packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/components/CreateCollection/CreateCollection.js

```diff
@@ -0,0 +1,81 @@
+import counterpart from 'counterpart'
+import { Box, Button, CheckBox, FormField, TextInput } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function CreateCollection ({
+  collection,
+  disabled,
+  onChange,
+  onSubmit
+}) {
+  const checkbox = React.createRef()
+  const textInput = React.createRef()
+  const { display_name, private: isPrivate } = collection
+  function updateCollection () {
+    const display_name = textInput.current.value
+    const isPrivate = checkbox.current.checked
+    onChange({
+      display_name,
+      private: isPrivate
+    })
+  }
+  return (
+    <React.Fragment>
```

Comment:
Perhaps this could be a `Box` with `tag="form"` prop? It seems that would be more semantic?

Reply:
Forms might require a method and an action. I’m not sure. There is a grommet form component, which we tried for the login form but had some problems with.

Reply:
Oh yeah, I thought they were going to drop `Form` for v2? I'm not so sure now. 

MDN says method and action can be defined on a button or input within the form instead: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

Reply:
I’m not sure what the action would be for these forms. Our API doesn’t accept form data so any action would have to be overridden on submit anyway, so it might not be important.

Reply:
Interesting that you can have buttons pointing to different URLs for the form data.